Automattic/jetpack
# Fix broken `@covers` annotations

#### Changes proposed in this Pull Request:

A few broken `@covers` annotations have shown up since the last time we did this. Fix them.

Also adjust Jetpack's configuration so it doesn't scan everything then exclude most of it.

#### Other information:


#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Check that the coverage job isn't complaining about invalid `@covers` anymore.
* Check that coverage is still being recorded properly for the Jetpack plugin.
